packages/genesys-spark-components/src/components/beta/gux-avatar-group/gux-avatar-group.tsx

```diff
@@ -0,0 +1,226 @@
+import {
+  Component,
+  h,
+  JSX,
+  Element,
+  State,
+  Prop,
+  writeTask
+} from '@stencil/core';
+import { afterNextRenderTimeout } from '@utils/dom/after-next-render';
+import { trackComponent } from '@utils/tracking/usage';
+import { GuxAvatarAccent } from '../gux-avatar/gux-avatar.types';
+import { randomHTMLId } from '@utils/dom/random-html-id';
+
+/**
+ * @slot - Some gux-avatars
+ */
+
+const MAX_AVATARS = 7;
```

Comment:
I think initially it would be good to have just a grouped component without it being responsive. 

I had been looking at making it responsive initially but resizing itself on resize was creating a loop that throttling, debouncing or using requestAnimationFrame couldn't fix. At this point, it was kind of getting to be a rabbit hole so I'll pause that effort until later and get the basic overflow working for the minute. 

I looked at other options using container queries or changing the type of slotted items, at this point, I think this is the simplest. Will continue to look at improving the add/remove from popup method. There's no simple way to use slotted content and reconfigure it from what I've found so far.